content/20.zksync-protocol/60.security/30.withdrawal-delay.md

```diff
@@ -53,9 +53,9 @@ On the other hand, **Internal Transactions** are initiated by a smart contract o
 Although these transactions can be prompted by user activity, they are not sent directly from one address to another but are instead part of the
 internal workings of a smart contract. Internal transactions may involve the transfer of ETH or ERC20 tokens between different addresses within the contract.
 
-Withdrawals from the ZKsync Era network are typically internal transactions managed by the
-[ZKsync Era Diamond Proxy](https://etherscan.io/address/0x32400084c286cf3e17e7b677ea9583e60a000324)
-contract. These transactions are recorded in the **Internal Transactions** section in Etherscan due to their internal nature.
+Withdrawals from any ZKsync Chain are typically internal transactions managed by the
+chain's Diamond Proxy contract ([see ZKsync Era's](https://etherscan.io/address/0x32400084c286cf3e17e7b677ea9583e60a000324)).
```

Comment:
Is this not the BridgeHub now (vs. Diamond Proxy)?

Reply:
fixed